docs/_includes/toc-layout.adoc

```diff
@@ -18,17 +18,19 @@ include::toc-left.adoc[]
 ====
 image::toc-left.png[Table of contents using left value]
 ====
-
-// NOTE: Setting `:toc: left` is equivalent to using the original AsciiDoc `toc2` attribute.
 // end::column[]
 
-// How does the toc-position attribute fit into this section?
-
 === Manual placement
 // tag::place[]
-The `macro` value on the toc attribute allows you to place the TOC anywhere in your document.
-To control where the TOC is rendered in your document, set the toc attribute to `macro`.
-Then, within the body of your document, use the `+toc::[]+` block macro to indicate the TOC's location.
+`toc` values of `auto`, `preamble`, and `macro` place the TOC in the body of the document.
```

Comment:
We should say "allow the TOC to appear in the main document area". Technically, the toc is always in the body of the document. We just move it around using the stylesheet. But that doesn't really matter to the user. It's all about how it appears.